src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js

```diff
@@ -169,10 +169,13 @@
     $(".page").hide();
     $("#stackpage").show();
 
-    $("#rawExceptionButton").click(function () {
+    $("#rawExceptionButton").click(function (event) {
```

Comment:
Did a little bit more research on this. You only need to `return false`. Remove the `event` and its `event.preventDefault()`. 

jQuery handles `return false` as do both `event.preventDefault()` AND `event.stopPropagation()`